### ISSUE TITLE
error: cannot find symbol class AudienceNetworkAds

### ISSUE THREAD

**Issue by KhoirH** [callstack/react-native-fbads#187]
I dont know why error import com.facebook.ads.AudienceNetworkAds in MyActivity.java
`
...
import com.facebook.ads.AudienceNetworkAds;
import com.reactnativenavigation.controllers.SplashActivity;
....
public class MainActivity extends SplashActivity {
   @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        AudienceNetworkAds.initialize(this);
    }
}
`

"react-native": "0.53.0"
"react": "16.0.0"
"react-native-navigation": "^1.1.484"

**Comment by KhoirH** [comment]
@Trancever yes, if me import audition-sdk manually in my gradle I will solve this problem

**Comment by kkusanagi** [comment]
Add this both. 

```
    implementation 'com.android.support:support-annotations:28.0.0' // Required Dependency by Audience Network SDK
    implementation 'com.facebook.android:audience-network-sdk:5.+'
```
